en-US/Conductors_Companion/03_Preparing_Quests_and_Adventures/Combat_Encounter_Building/Running_Hordes.md

```diff
@@ -0,0 +1,68 @@
+
+#### Determine Targets
+
+The circumstances of the encounter dictate how many members of the horde can attack the characters.
+Unless the circumstances dictate otherwise, assume the horde evenly spreads its attacks across all characters.
+If certain characters step ahead of the rest of the party or block choke points that prevent the horde from reaching other characters, you can redirect the horde's attacks to the characters stepping forward.
+
+#### Adjudicating Attacks and Saving Throws
+
+Whenever rolling individual attacks or saving throws would be a burden, assume that one quarter of attacks or saving throws rolled by the horde succeed.
+Round up or down depending on the circumstances, such as when determining how many attacks succeed against characters with wildly different ACs.
+
+If all the creatures in a horde have advantage on an attack or saving throw, increase the number of successful attacks or saving throws to one half.
+If the horde has disadvantage, reduce the number to one in ten.
+
+If any member of the horde is affected by an effect that leaves them incapacitated, remove them from play.
+
+If you prefer to roll dice, roll twice when a group of monsters all make attacks or saving throws.
+On each success, one quarter of the monster attacks or saving throws succeed.
+If both rolls fail, no attacks or saving throws succeed.
+
+#### Adjudicating Areas of Effect
+
+Adjudicate the number of creatures caught up in an area of effect based on the circumstances, but leaning toward more creatures rather than fewer.
+You can use the following as a baseline for the number of tightly packed creatures in a horde that are affected in a given area:
+
+- Tiny Area (5-foot radius): Two creatures
+- Small Area (10- to 15-foot radius): Four creatures
+- Large Area (20-foot radius): Sixteen creatures
+- Huge Area (30-foot radius or more): Thirty-two or more creatures
+- Short Line (60 feet): Six creatures
+- Long Line (120 feet): Eight creatures
+
+<!-- style:sidebar -->
+> **Horde Tips and Tricks** \
+> It is helpful to keep the following in mind when running hordes:
+>
+> - Describe these horde combat rules to the players so everyone understands how they work.
+>   Always adjudicate combat to the characters' benefit, and help the players achieve their goals.
+> - Use evocative descriptions to flavor a horde.
+>   Worry less about the mechanics and more about the feeling involved in fighting a huge horde of monsters.
+> - Avoid using hordes with monsters of different types in a single battle.
+>   Instead, use a single stat block and describe any physical differences between monsters narratively.
+> - Augment a horde with a handful of more powerful monsters when desired.
+>   Track the hit points and attacks of these powerful monsters normally.
+> - As hordes diminish to a manageable level, return to tracking individual hit points, attacks, and saving throws normally.
```

Comment:
I added this as a sidebar, since it seemed to make more sense like that than as a subsection. Do you agree?